S2VX.Game.Tests/S2VXTestBrowser.cs

```diff
@@ -1,17 +1,30 @@
+using osu.Framework.Allocation;
 using osu.Framework.Graphics;
-using osu.Framework.Graphics.Cursor;
 using osu.Framework.Platform;
 using osu.Framework.Testing;
 
 namespace S2VX.Game.Tests {
+    /// <summary>
+    /// Tests ran  through the osu!framework visual test runner use this class.
+    /// Tests ran through Test Explorer or the GitHub pipeline do not use this
+    /// class.
+    /// </summary>
     public class S2VXTestBrowser : S2VXGameBase {
+        /// <summary>
+        /// S2VXStory resolves an S2VXCursor as one of its dependencies. To
+        /// avoid having to manually cache a Cursor into many visual tests, we
+        /// can cache it here.
+        /// </summary>
+        [Cached]
+        private S2VXCursor Cursor { get; set; } = new();
```

Comment:
We can move this to S2VXGameBase so we don't need to create and cache it here AND in S2VXGame

Reply:
I remember testing this yesterday without great results because of the way S2VXGameBase sets its Content. I'll try a few thing more tonight to confirm.

Reply:
From what I tested, I plan to keep the changes as is. I tried moving `Cursor` into `GameBase`, but ran into issues because of the way the container's content is being set. I also tried moving it into `SquareContainer`, but ran into problems with the load order of the dependencies.

Reply:
Moved to S2VXGameBase